.github/workflows/main.yaml

```
@@ -226,7 +227,7 @@ jobs:
       - run: |
           export PROTOCOL=${{ matrix.cases.protocol }}
           export KIND_IMAGE=kindest/node:${{ matrix.cases.version }}
-          export CONTAINER_RUNTIME="remote"
+          export CONTAINER_RUNTIME=${{ matrix.container-runtime }}
```

```suggestion
          export CONTAINER_RUNTIME="containerd" 
```

and remove the new line at Line 179, for we still want to test with containerd in previous e2e_test 😃